src/styles/saas-list.less

```diff
@@ -1,76 +1,104 @@
+.build-applications-view {
+
+  h1 {
+    color: @saas-text;
+    margin: 0;
+    padding: 20px;
+    padding-left: (@grid-gutter-width / 2);
+    width: 100%;
+  }
+}
+
 .saas-offerings-container {
-  background: @color-pf-black-200;
-  padding-left: 0px;
 
   .spinner-container {
     padding-top: 60px;
     padding-bottom: 60px;
   }
 
-  .card-view-pf {
-    padding-left: 21px;
-  }
-
-  .card-view-pf .card-content {
-    background-color: @color-pf-white;
-    cursor: pointer;
-    margin: 0;
+  .saas-list {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    margin: 0 (@grid-gutter-width / 2);
 
     @media (min-width: @screen-sm-min) {
-      margin: 2px 0 10px;
+      flex-direction: row;
     }
-  }
 
-  .card-view-pf .card {
-    height: 100%;
-    width: 100%;
-    cursor: pointer;
-    text-align: inherit;
-    margin-bottom: 8px;
+    .card {
+      align-items: center;
+      border-bottom: solid 1px @saas-text;
+      color: @saas-text;
+      cursor: pointer;
+      display: flex;
+      height: 5em;
+      width: 100%;
 
-    @media (min-width: @screen-sm-min) {
-      height: 200px;
-      width: 260px;
-      margin-bottom: 20px;
+
+      &:hover {
+        background-color: @saas-hover;
+      }
+
+      @media (min-width: @screen-sm-min) {
+        border-bottom: none;
+        flex: 0 1 auto;
+        height: 300px;
```

Comment:
One of the benefits of flexbox is that siblings with automatically assuming the same height as one another.  So it's probably best if you don't set a fixed height.